api/lib/models/checkout-items/checkout-items-data-mapper-impl.ts

```diff
@@ -142,23 +131,23 @@ export class CheckoutItemsDataMapperImpl extends GenericDataMapper
       autoQuoteFieldNames: false,
       autoQuoteTableNames: false,
     })
-      .from('CHECKOUT_DATA', 'c') // Here
+      .from(this.checkoutObjectDataMapperImpl.tableName, 'c')
       .field('COUNT(uid)')
       .where('c.item_id=i.uid')
-      .toString();
+      .toParam();
     const query = squel.select({
       autoQuoteFieldNames: false,
       autoQuoteTableNames: true,
     })
       .fields([`i.quantity - (${subquery}) AS available`, 'i.*'])
```

Comment:
this needs to be subQuery.text as we're calling toParam() on this. Also concatenate the two arrays for the parameters below. So `query.values = subquery.values.concat(query.values);`
```suggestion
      .fields([`i.quantity - (${subquery.text}) AS available`, 'i.*'])
```